components/look-at/index.js

```diff
@@ -81,9 +82,18 @@ AFRAME.registerComponent('look-at', {
     // Track target object position. Depends on parent object keeping global transforms up
     // to state with updateMatrixWorld(). In practice, this is handled by the renderer.
     var target3D = this.target3D;
+    var object3D = this.el.object3D;
     if (target3D) {
-      var vector = this.el.object3D.parent.worldToLocal(target3D.getWorldPosition());
-      return this.el.object3D.lookAt(vector);
+      var target = object3D.parent.worldToLocal(target3D.getWorldPosition());
+      var vector;
+      if (this.isCamera) {
+        // look away from target.
+        vector = new THREE.Vector3();
```

Comment:
I would create a helper vector in the `init` handler, reuse it, and not create a new one each frame